generators/chipyard/src/main/scala/HarnessBinders.scala

```diff
@@ -83,7 +83,9 @@ class WithGPIOTiedOff extends OverrideHarnessBinder({
 // DOC include start: WithUARTAdapter
 class WithUARTAdapter extends OverrideHarnessBinder({
   (system: HasPeripheryUARTModuleImp, th: HasHarnessSignalReferences, ports: Seq[UARTPortIO]) => {
-    UARTAdapter.connect(ports)(system.p)
+    withClockAndReset(th.harnessClock, th.harnessReset) {
+      UARTAdapter.connect(ports)(system.p)
```

Comment:
Was this necessary for correct behavior? Curious

Reply:
Technically no... but in a previous commit there was a difference between the implicit harness clock/reset and the `th.harness<Clock/Reset>`. This should be fixed now so I reverted this.